randovania/games/samus_returns/exporter/joke_hints.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+JOKE_HINTS = [
+    "This hint has been sealed away.",
+    "Spidersparking can quickly cover large distances.\nTake advantage whenever possible!",
+    "The Spider Boost can be used while in Spider Ball form.\n"
+    "It will launch you forward, damaging enemies you make contact with.",
+    "Drag pins onto the map to mark points of interest.",
+    "This hint hasn't returned yet.",
```

Comment:
see sealed away text

Reply:
`Samus, we cannot return this hint to you.`